crates/sncast/tests/helpers/constants.rs

```diff
@@ -15,4 +15,4 @@ pub const DEVNET_OZ_CLASS_HASH_CAIRO_0: &str =
 pub const DEVNET_PREDEPLOYED_ACCOUNT_ADDRESS: &str =
     "0x691a61b12a7105b1372cc377f135213c11e8400a546f6b0e7ea0296046690ce";
 pub const DEVNET_OZ_CLASS_HASH_CAIRO_1: &str =
-    "0x61dac032f228abef9c6626f995015233097ae253a7f72d68552db02f2971b8f";
+    "0x4c6d6cf894f8bc96bb9c525e6853e5483177841f7388f74a46cfda6f028c755";
```

Comment:
Note: It will be necessary to update it after devnet updates.

Reply:
let's add in code comment with a path where it is defined in devnet, maybe?

Reply:
> This needs to be updated whenever we update the devnet

is this true? I think class hashes are idependent of devnet version, and we will need to update this only if devnet updates OZ contract, but let's double-check

Reply:
> we will need to update this only if devnet updates OZ contract

This is correct. However, it looks like class hashes are actually updated with each devnet version. For example compare [v0.0.2](https://github.com/0xSpaceShard/starknet-devnet-rs/blob/v0.0.2/crates/starknet-devnet-core/src/constants.rs#L20) and [v0.0.3](https://github.com/0xSpaceShard/starknet-devnet-rs/blob/v0.0.3/crates/starknet-devnet-core/src/constants.rs#L21)

Reply:
Perhaps, but we should not expect this to happen often

Reply:
I've removed that comment, left only the relevant link to the devnet